src/graphnet/models/detector/icecube.py

```diff
@@ -40,6 +40,75 @@ def _forward(self, data: Data) -> Data:
 
         return data
 
+class IceCube86_v2(Detector):
```

Comment:
We probably ought to make it clear that this takes in different data than `IceCube86` and therefore is not just a version/alternative of this.
```suggestion
class IceCube86_DOM(Detector):
```

Reply:
Yeah, let's do that. But at the same time, I think it's a little unintuitive to give this a different "detector" class. Perhaps we should rethink this structure a little in the future.